coeus-impl/src/main/resources/org/kuali/coeus/propdev/impl/budget/subaward/ProposalBudgetSubawardsPage.xml

```diff
@@ -138,7 +141,7 @@
 					p:order="10" />
 				<bean parent="PropBudget-SubawardsAttachments-FileUpload-Section" 
 					p:hidden="@{dialogDataObject?.budgetSubAwardFiles == null ? false : !dialogDataObject.budgetSubAwardFiles.isEmpty()}" 
-					p:order="20" />
+					p:order="20" p:render="@{!#parent.readOnly}"/>
```

Comment:
May be a stupid question, but why (in readonly mode) hide the file upload section here, but above you show a disabled save button? Is that a standard we are following?